rero_ils/modules/ext.py

```diff
@@ -112,23 +112,22 @@ def register_import_api_blueprint(self, app):
             __name__
         )
         endpoints = app.config.get('RERO_IMPORT_REST_ENDPOINTS', {})
-        for endpoint, options in endpoints.items():
+        for endpoint in endpoints:
             imports_search = ImportsListResource.as_view(
-                f'import_{endpoint}',
-                import_class=options.get('import_class'),
-                import_size=options.get('import_size')
+                f'''import_{endpoint.get('key')}''',
```

Comment:
Maybe better to write `f'import_{endpoint.get("key")}'`
See also endpoints definition.

Reply:
+1